Spacegray.sublime-theme

```diff
@@ -544,11 +565,190 @@
         "settings": ["bold_folder_labels"],
         "font.bold": true
     },
-    // Sidebar entry selected
+
+//
+// SIDEBAR - VCS
+//
+
+    // Decolor default VCS badges
+
+    {
+        "class": "vcs_status_badge",
+        "settings": ["vcs_color"],
+        "layer0.tint": "var(base02)"
```

Comment:
Is the point of the setting that it only colors the text and not the icon? Seems a bit weird to be, but since it's optional I don't really mind.

However, that also means that there's no option currently to enable both, text and icon coloring, right?

Reply:
Thank you for taking a look!

I think I did this as to not have anything colorful next to the gutter, otherwise it can get noisy, and gutter's signals are more important. The icons aren't decolored when the VCS integration isn't on because otherwise it becomes hard to parse the sidebar for VCS info.

Toxin, the main inspiration, hid the icons completely, for example.

> However, that also means that there's no option currently to enable both, text and icon coloring, right?

Correct. Can be added later if somebody wants it, what do you think?

Reply:
Indeed, this is not a blocker, although when making this configurable separately I would probably choose different setting names to more clearly indicate which part of the siedebar they are controlling.

Reply:
Ah, setting names were adapted from Toxin, but it can be changed, since that theme is all but obsolete. What do you propose?

Reply:
I'm thinking in the lines of "vcs_sidebar_label", leaving the option open for another "vcs_sidebar_icon" setting.

Reply:
Something odd is going on with GitHub (contacted support about it), the branch isn't updating, but in the meanwhile, check [this diff](https://github.com/alecmev/spacegray/commit/8fa0a05c808e03bc4fcab0b7488c6ff1291a25d4), please, and let me know if it looks good to you.

Reply:
That looks perfect to me, thanks for making the adjustments. Feel free to ping me when the change arrived at this PR or alternatively just open a new one.

Reply:
Ping! 😉